platform-core-module/src/main/java/com/softicar/platform/core/module/file/smb/SmbIOException.java

```diff
@@ -0,0 +1,28 @@
+package com.softicar.platform.core.module.file.smb;
+
+import com.softicar.platform.common.core.exceptions.SofticarIOException;
+import java.io.IOException;
+
+/**
+ * Thrown when an {@link IOException} occurs while an {@link ISmbClient}
+ * implementation interacts with an SMB share.
+ *
+ * @author Alexander Schmidt
+ */
+public class SmbIOException extends SofticarIOException {
+
+	public SmbIOException(IOException ioException) {
+
+		this(ioException, "An I/O exception occured while using an SMB share: %s", ioException);
```

Comment:
is this really a good practice to concat the causing exception message? We should rather concat the share URL here, IMO.

Reply:
Dunno - I just replicated this pattern from the base class, `SofticarIOException`.
Btw, we don't have a share URL in all places where this Exception is throws.

Reply:
not satisfied, but we have more important things to do